contents/tutorials/event-tracking-guide.md

```diff
@@ -2,184 +2,181 @@
+## Autocapture’s limitations
 
-The relationship between events and actions can be both one-to-one or many-to-one. What this means is that you can create an action from one event only, such as a "Pageview" action for a `$pageview` event, as well as an action targeting multiple events, like a "Sign Up" action targeting the events `clicked button with text "Sign Up"` and `clicked button with text "Sign Up Now"`.
+Although we are proud of our autocapture feature, we know it isn’t enough for more advanced product analytics. This is for two reasons.
 
-Not only that, you can also create actions based on the URL where an event happened. This could lead to actions such as "Clicked Sign Up on About Page".
+First, event autocapture can be overwhelming. In high-volume instances, where events come in at a high rate, the live events table can contain so much information it can be difficult to determine exactly what you care about. Remember that every click, change of input, or submission is tracked here. Without proper filters, actions, and insights set up, this data can be overwhelming.
 
-There are two main reasons for creating actions:
+Second, autocapture is a general solution that provides value to many users. To get the most out of PostHog, you should be using custom events. Custom events let you track *exactly* the behavior you care about, ranging from a user hovering an input, to a function call in your backend.
 
-#### 1. Tagging events that you especially care about**
+## Setting up custom events
 
-Filter out the noise from all your events to make it easier for you to execute your analytics processes.
+To expand on autocapture, you can create and add custom events to your app. Custom events allow you to capture details from anywhere in the codebase whether it is a button press on the frontend or a class method call on the backend. This ensures correct and comprehensive data capture.
 
-#### 2. Combining multiple events into one action**
+To set up custom events, first, you need to install the library for the language or framework you are using. We have built libraries for a range of languages and frameworks including Javascript, Python, iOS, Android, and more. You can find our [full list of libraries here](https://posthog.com/docs/integrate#libraries).
 
-Let's say you have multiple sign up buttons spread across your website. For some types of analysis it could be useful to know which specific button was clicked, however, it might also be the case that you just care that the user signed up, irrespective of what they clicked to do so.
+For example, with Python (and Python frameworks like Django or Flask), setting up custom events starts with installing the PostHog package with pip (or your package manager of choice). 
 
-Actions fit this use-case perfectly, since events are combined using `OR` operations. That means that if an action is composed of multiple events, it will be registered if _any of the events composing it_ are triggered. 
```

Comment:
it'd be cool to use the same snippet system we use in group analytics for example, so we have it for every language
https://posthog.com/docs/user-guides/group-analytics

Reply:
Awesome, good catch 👍